ErlangInstallerPreferences/ReleasesTableViewDelegate.swift

```diff
@@ -19,7 +19,7 @@ class ReleasesTableViewDelegate: NSObject, NSTableViewDelegate, NSTableViewDataS
         let release = ReleaseManager.available[rowIndex]
         
 		if UserDefaults.defaultRelease == release.name {
-			cellView.releaseNameLabel.font =  NSFont.boldSystemFont(ofSize: NSFont.systemFontSize())
+            cellView.releaseNameLabel.font = NSFont.boldSystemFont(ofSize: NSFont.systemFontSize())
```

Comment:
We should stick to either tabs or spaces.

Reply:
spaces are better IMHO. I forgot to AutoIndent